src/clr-angular/timeline/_variables.timeline.scss

```diff
@@ -1,8 +1,11 @@
 // colors
-$clr-timeline-line-color: $clr-color-neutral-500; // #9a9a9a
-$clr-timeline-step-header-color: $clr-color-neutral-600; // #737373
-$clr-timeline-step-title-color: $clr-color-neutral-700; // #565656
-$clr-timeline-step-description-color: $clr-color-neutral-700; // #565656
+$clr-timeline-line-color: $clr-color-neutral-500 !default; // #9a9a9a
+$clr-timeline-step-header-color: $clr-color-neutral-600 !default; // #737373
+$clr-timeline-step-title-color: $clr-color-neutral-700 !default; // #565656
+$clr-timeline-step-description-color: $clr-color-neutral-700 !default; // #565656
+$clr-timeline-step-current-color: $clr-action-blue !default;
+$clr-timeline-step-complete-color: $clr-green !default;
+$clr-timeline-step-error-color: $clr-red !default;
 
```

Comment:
😱 
I forgot that I had to add these on my topic branch.

So that we don't get into a conflict/deprecation hell, could you update them to the following?

```
// colors
$clr-timeline-line-color: $clr-color-neutral-500; // #9a9a9a
$clr-timeline-step-header-color: $clr-color-neutral-600; // #737373
$clr-timeline-step-title-color: $clr-color-neutral-700; // #565656
$clr-timeline-step-description-color: $clr-color-neutral-700; // #565656

$clr-timeline-incomplete-step-color: $clr-color-neutral-600;
$clr-timeline-current-step-color: $clr-color-action-600;
$clr-timeline-success-step-color: $clr-color-success-400;
$clr-timeline-error-step-color: $clr-color-danger-800;
```

Reply:
Will add.